sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -68,7 +63,7 @@ function getTopicFilterOrUndefined(value: any): SqlFilter | CorrelationFilter |
       compatibilityLevel: getIntegerOrUndefined(value["CompatibilityLevel"]),
       requiresPreprocessing: getBooleanOrUndefined(value["RequiresPreprocessing"])
     };
-  } else if (value["CorrelationId"] != undefined) {
+  } else {
     result = {
       correlationId: getStringOrUndefined(value["CorrelationId"]),
```

Comment:
Are we saying `CorrelationId` is always defined now? Should this be using `getString` then?

Reply:
No, it's not always defined for CorrelationFilters as it's just another property that maybe checked on. Hence, updated it to use `getStringOrUndefined`